frontend/whiteboard-app/src/components/WhiteBoardComponent.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import WhiteboardPage from './WhiteboardPage'
+
+const WhiteBoardComponent = () => {
+  return (
+    <div className='whiteBoardPageParent'>
+      <WhiteboardPage />
```

Comment:
Wie im Kommentar zur App.tsx: die WhiteboardComponent ist in der Page enthalten. Die WhiteboardComponent ist NUR das Whiteboard selbst. Auf der Page gibt es auch noch einen Header und Buttons etc